File: src/test/bench/fib/fib.lua
local start = os.clock()
local function fib(n)
    if n < 2 then return n end
    return fib(n - 2) + fib(n - 1)
end

local res = fib(30)
io.write("time: ", (os.clock() - start) * 1000, "\n")
io.write(res, "\n")

File: src/test/bench/fib/fib.c
#include <stdio.h>
#include <time.h>

int fib(int n) {
    if (n < 2) {
        return n;
    }
    return fib(n-1) + fib(n-2);
}

int main() {
    float startTime = (float)clock()/CLOCKS_PER_SEC;
    int res = fib(30);
    float endTime = (float)clock()/CLOCKS_PER_SEC;
    printf("time: %f\n", (endTime - startTime) * 1000);
    printf("%d\n", res);
    return 0;
}
